ui/src/app/cohort-search/selection-list/selection-list.component.tsx

```diff
@@ -164,26 +167,34 @@ export class SelectionList extends React.Component<Props> {
   }
 
   render() {
-    const {back, close, disableFinish, finish, removeSelection, selections, setView} = this.props;
+    const {back, close, disableFinish, finish, removeSelection, criteria, selections, setView} = this.props;
     return <div style={styles.selectionPanel}>
       <h5 style={styles.selectionTitle}>Selected Criteria</h5>
-      <div style={styles.selectionContainer}>
-        {selections.map((selection, s) =>
+      {serverConfigStore.getValue().enableCohortBuilderV2 && criteria && <div style={styles.selectionContainer}>
+        {criteria.map((selection, s) =>
           <SelectionInfo key={s}
             index={s}
             selection={selection}
             removeSelection={() => removeSelection(selection)}/>
         )}
-      </div>
-      <div style={styles.buttonContainer}>
+      </div>}
+      {!serverConfigStore.getValue().enableCohortBuilderV2 && <div style={styles.selectionContainer}>
+        {selections.map((selection, s) =>
+            <SelectionInfo key={s}
+                           index={s}
+                           selection={selection}
+                           removeSelection={() => removeSelection(selection)}/>
+        )}
+      </div>}
+      {!serverConfigStore.getValue().enableCohortBuilderV2 && <div style={styles.buttonContainer}>
         <Button type='link'
           style={{...styles.button, color: colors.dark, fontSize: '14px'}}
           onClick={() => close()}>
           Cancel
         </Button>
         {this.showNext && <Button type='primary' onClick={() => setView('modifiers')}
           style={styles.button}
-          disabled={selections.length === 0}>
+          disabled={!criteria || criteria.length === 0}>
```

Comment:
Need to revert this change also, the Finish button stays disabled after selections are made in the non feature-flagged UI